src/chi/components/popover/popover.scss

```
@@ -203,4 +203,47 @@ $popover-arrow-size: 0.75rem;
       }
     }
   }
+
+  &.-closable {
+    position: relative;
+
+    .chi-popover__content {
+      padding-top: 2.5rem;
+    }
+
+    header {
+      &.chi-popover__header {
+        padding-right: 2.5rem !important;
```

`!important` does not appear to be necessary and it will prevent padding utility classes from working. Can we remove?

```suggestion
        padding-right: 2.5rem;
```